bin/run-concurrency-test

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 set -euo pipefail
 
+if [ $# -ne 1 ]; then
+  echo "usage: $0 FILENAME.sh"
+  exit 1
+fi
+
+source "$1"
+export HTTP_TEST_{LATENCY_{DISTRIBUTION,MEAN,NORMAL_{MEAN,STDDEV},EXPRESSION_{MEAN,STDDEV}_MS},RATE_LIMIT_{BEHAVIOR,HARD_STATUS_CODE,BUCKET_{CAPACITY,QUANTUM,FILL_INTERVAL}}}
```

Comment:
Maybe we could do:

```bash
set -o allexport
source $1
set +o allexport
```

?

That should save us from having to modify this if we add additional env vars.

Reply:
Oh, that's good! I hadn't seen that one before, and definitely exactly what we need.